headers.go

```diff
@@ -157,6 +165,17 @@ func (h ProtectedHeader) ensureCritical() error {
 	return nil
 }
 
+// ensureCritical ensures IV and Partial IV are not both present in the protected bucket.
+func (h ProtectedHeader) ensureIV() error {
+	if _, ok := h[HeaderLabelIV]; !ok {
+		return nil
+	}
+	if _, ok := h[HeaderLabelPartialIV]; !ok {
+		return nil
+	}
+	return errors.New("The 'Initialization Vector' and 'Partial Initialization Vector' parameters must not both be present in the protected header")
```

Comment:
All the other errors in this file are prefixed with `cbor:` also generally Go errors are lower case on the first letter. Is the differentiation meaningful here?